src/Grpc/test/InteropTests/Infrastructure/ClientProcess.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Diagnostics;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Internal;
+using Xunit.Abstractions;
+
+namespace InteropTests.Infrastructure
```

Comment:
Woah, this is the start of a lot of new test infrastructure that doesn't seem to come from the gRPC repo. Don't we have anything similar already?

Reply:
I reuse ProcessEx. ClientProcess and WebServerProcess are help types on top of that.